packages/shared/src/tableDefinitions/promptsTable.ts

```diff
@@ -77,3 +74,7 @@ export function promptsTableColsWithOptions(
     return col;
   });
 }
+
+export function webhookActionFilterOptions(): ColumnDefinition[] {
+  return promptsTableCols.filter((col) => col.id === "name");
+}
```

Comment:
This seems super odd as a util function. Do we expect to support more columns at some point?